README.md

```diff
@@ -6,29 +6,31 @@ carbonite-migrate
 User requirements
 =================
 
-**The following software must be running/installed on the client computer**:
+**The following software must be running/installed on the user's computer**:
 
-1.  Device42 16.10.00+
+1. Device42 16.10.00+
 
-2.  Carbonite console 8.3.0.293+
+2. Carbonite console 8.3.0.293+
 
-3.  **After installing the Carbonite Console, please copy the
+3. Carbonite Migrate — The source server must have Carbonite Migrate installed and licensed on it.
+
+4. **After installing the Carbonite Console, please copy the
     "DoubleTake.PowerShell.dll" from its install directory (by default in
     C:\\Program Files\\Carbonite\\Replication\\Console) to the root folder of
     your scripts' location!**
 
-4.  PowerShell 5.1: if you are using Windows 10, you already have the necessary
+5. PowerShell 5.1: if you are using Windows 10, you already have the necessary
```

Comment:
Should specify this is needed on the computer that the PowerShell scripts are being run on.

Reply:
Done